Add eurucamp 2015

zhephyn opened the pull request:
This PR adds the 2015 edition of eurucamp. 
Fixes #715

marcoroth commented:
@zhephyn for the 2014 one, you can add them using the "external player", check out Brighton Ruby 2015 for reference:

https://www.rubyevents.org/talks/making-software-fun
https://github.com/rubyevents/rubyevents/blob/main/data/brightonruby/brightonruby-2015/videos.yml

marcoroth commented on data/eurucamp/eurucamp-2015/videos.yml:
```diff
@@ -1,6 +1,402 @@
+    Ruby is a great language for building CLIs. There’s an amazing selection of existing libraries like Thor, GLI, or even OptionParser. You probably use a number of Ruby tools everyday like chef, the heroku toolbelt, , and of course the rails command line tool.
+    Though it’s easy to build a Ruby CLI, distributing it to end users is much more complicated because it requires a Ruby runtime. One option is to rely on Ruby being already installed on the end user’s computer. The other is to bundle ruby as part of the distribution package. Both of these are problematic so much that Vagrant, CloudFoundry and hub (Github command-line tool) have all switched ov ...
+    But there’s still hope! In 2012, Matz started working on a lightweight embeddable ruby called mruby. Since mruby is designed for being embedded in other compiled languages, it also solves the distribution problem. In this talk we’ll look how we can write CLI tools using mruby to produce a self-contained binary that can be shipped to end users.
+
+    Help us caption & translate this video!
+
+    http://amara.org/v/GxRr/
```
```suggestion
```

marcoroth commented on data/eurucamp/eurucamp-2015/videos.yml:
```diff
@@ -1,6 +1,402 @@
+    The Junior Jump by Rebecca Poulson
+
+    The Junior Jump deals with a topic that is increasingly important to the Ruby community--onboarding new engineers into their first jobs. We’ll discuss how the interview stage can be used to design a roadmap for a new engineer’s early days, how to select meaningful and appropriate projects and how to come up with a mentorship model that works given the resources at your organization. We will h ...
+
+    Help us caption & translate this video!
+
+    http://amara.org/v/GxRq/
```
```suggestion
```

marcoroth commented on data/eurucamp/eurucamp-2015/videos.yml:
```diff
@@ -1,6 +1,402 @@
+    Programming Parents by Daniel Schweighöfer
+
+    Working as a developer while raising a child isn’t as difficult as you may think. A big share of adults are parents, and many can still become one some day. Becoming a parent changes a lot of things in your life. What it doesn’t change is your ability to work. It surely changes your priorities, and you don’t get much sleep during the first years, but this doesn’t have to prevent anybody from  ...
+
+    Help us caption & translate this video!
+
+    http://amara.org/v/GxRp/
```
```suggestion
```

marcoroth commented on data/eurucamp/eurucamp-2015/videos.yml:
```diff
@@ -1,6 +1,402 @@
+    But what does technical actually mean? What makes someone technical? What makes them not technical?
+    In this talk, the speaker will explore how “not technical” is dangerous phrase we use to ensure that we need never question our hidden biases about others and their aptitudes. She will demonstrate how this phrase not only encourages a lack of collaboration, but leads to stagnation in the growth of your orgnization by actively discouraging your peers from learning new skills.
+    She will conclude with concrete steps attendees can take to better understand the capabilities of everyone on their team – coders or not – so their projects, company or teams can be most successful.
+
+    Help us caption & translate this video!
+
+    http://amara.org/v/GxRn/
```
```suggestion
```

marcoroth commented on data/eurucamp/eurucamp-2015/videos.yml:
```diff
@@ -1,6 +1,402 @@
+
+    Since the inception of the programmable computer we have grappled with the challenge of validating the outcome of a conversation in code between a human and a machine. Programming as a performance embraces this challenge, putting it at the centre of a performance. Focusing on the liveliness and complexity of dancing with a machine in realtime. In a performance a human edits and writes algorit ...
+    This talk will dive into the world of the live programmer and how you can turn your Ruby coding skills into a live performance. Discover what performance and expression has to teach us about programming.
+
+    Help us caption & translate this video!
+
+    http://amara.org/v/GxS1/
```
```suggestion
```

marcoroth commented on data/eurucamp/eurucamp-2015/videos.yml:
```diff
@@ -1,6 +1,402 @@
+  video_provider: "youtube"
+  video_id: "U_yIxD_U-IY"
+
+- title: "Lightning Talks"
+  raw_title: "eurucamp 2015 - Lightning Talks"
+  speakers:
+    - "Isaiah Peng"
+    - Ruby Corns study group
+  date: "2015-08-02"
+  event_name: "eurucamp 2015"
+  published_at: "2015-08-14"
+  announced_at: "2014-11-17"
+  description: |-
+    Protocol buffers on JRuby by Isaiah Peng
+
+    Study Groups by Ruby Corns
+
+    Ruby Issues
+
+    Shoes
+  video_provider: "youtube"
+  video_id: "mJcxAY172y8"
```
Would you mind adding these 4 lightning talks as subtalks for the "Lightning Talks" video? Thank you!

marcoroth reviewed (commented):
Thank you for working on this @zhephyn! :pray:

zhephyn commented:
@marcoroth So i added the 2014 edition and resolved all the commit suggestions you had mentioned earlier. 

Only issue I'm having is getting the video thumbnail to show here instead of "This talk is available on AWS". Not sure how to solve that. 
<img width="1312" height="677" alt="image" src="https://github.com/user-attachments/assets/d536c810-2568-467f-86bb-3114f82c4d37" />

Surprisingly, when i click on the "Play on Website button", it takes me to a browser session and the real video plays without issues. 
<img width="1294" height="667" alt="image" src="https://github.com/user-attachments/assets/3751730a-5b29-403b-a823-f57455d29e65" />


Also, i added the child talks for the lightning talk. However, now the CI is failing because the thumbnails for these child talks couldn't be generated. Not sure how to go about that as well.
```ruby
 missing thumbnail for child_talk: polar-buffers-lightning-talk-eurucamp-2015 at: /home/runner/work/rubyevents/rubyevents/app/assets/images/thumbnails/polar-buffers-lightning-talk-eurucamp-2015.webp
missing thumbnail for child_talk: study-groups-lightning-talk-eurucamp-2015 at: /home/runner/work/rubyevents/rubyevents/app/assets/images/thumbnails/study-groups-lightning-talk-eurucamp-2015.webp
```

zhephyn commented:
@marcoroth. Never mind my last comment. 

I've fixed every issue that the PR had including the changes related to sub talks for the lightning talk. I also added thumbnails for the 2014 edition. 

kindly take a look when its convenient for you. Thanks!

marcoroth reviewed (approved):
Thank you @zhephyn! 🙏🏼
